pkg/build/graph/helpers.go

```diff
@@ -76,3 +78,31 @@ func defaultNamespace(value, defaultValue string) string {
 	}
 	return value
 }
+
+// BuildConfigForTag returns the buildConfig that points to the provided imageStreamTag.
+// TODO: Handle multiple buildconfigs pointing to the same tag.
```

Comment:
Should we make an issue for this? I know I forget about `//TODO`s really easily. We can add it to a list of things we'd like to have done before exposing the graph API, if there is a list somewhere.